Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -370,6 +370,29 @@ public class Fuzzer {
         return execution
     }
 
+    private func recollectStaticTypes(for program: Program) {
```

Comment:
Maybe call it `inferMissingTypes` or something like that? I think "collect" is a bit confusing here